cmd/otelcontribcol/Dockerfile

```diff
@@ -1,15 +1,25 @@
 FROM alpine:latest as prep 
 RUN apk --update add ca-certificates
 
+ARG USER_UID=10001
+# github.com/containers/podman/v2 requires these directories to be present
+# and be writable. When this is not satisfied, the library raises a fatal
+# error and crashes the collector.
+# This is required by the podmanreceiver.
+RUN mkdir -p /rundir
+RUN mkdir -p /tmp
+RUN chown -R ${USER_UID}:${USER_UID} /rundir
```

Comment:
As already said, work on Dockerfile and introduce kind of complexity will probably introduce issues between containers runtimes and OS.
If tomorrow we provide an image for windows container (PR already exists) as an exemple, these steps will not be compatible, while just copying otelcontribcol is.

Reply:
the dockerfile for windows would be a separate file yea? so it would affect the image for windows.

Reply:
Hey @gillg thanks. Just to clarify are you referring line 8-11 in the Dockerfile or changes related to journalctl?

Reply:
Not 100% sure if windows will be a different file. The image can be just an ARG and buildkit can create all flavors.
And my comment is pretty general but just trigger here last changes related to adaptations made in podman context.